pandas/io/excel/_base.py

```diff
@@ -541,7 +541,7 @@ class ExcelWriter(metaclass=abc.ABCMeta):
 
     Parameters
     ----------
-    path : str
+    path : str or BinaryIO
```

Comment:
`io.BytesIO`? What do we actually need here?

Reply:
`io.Bytes` doesn't fully cover possibilities of ExcelWriter, as I found `BinaryIO` from typeshed is best option, Pycharm doesn't show any warnings

Reply:
You should have `typing.BinaryIO` here then.

Reply:
Done